### ISSUE TITLE
Will you release the fine-tuned weights?

### ISSUE THREAD

**Issue by XiaohuJoshua** [gyxxyg/VTG-LLM#21]
Thanks for the impressive work. I tested some videos in qvhighlights val split and follow the prompt: "Find the video segment that corresponds to the given textual query '{}' and determine its start and end seconds." The predicted duration in the response can be random sometimes. (e.g, 0136.8-0143.4s for this try,  0146.5-0151.8s for another try). Which affects the practicality of the model. I guess it could be the model has not been fine-tuned to fit qvhighlights' distribution. So, do you have the plan to release the fine-tuned weights.

**Comment by XiaohuJoshua** [comment]
Or could you offer some good examples for moment retrieval with pretrained weights? Thanks.

**Comment by gyxxyg** [comment]
Thank you for your attention! At present, we have not fine-tuned the models for the moment retrieval task on QVHighlights; we have only tested them for the highlight detection task. As our schedule is quite packed lately, we will fine-tune the models at a later time. Please stay tuned for updates.

By the way, we have discovered that reducing the temperature significantly improves the grounding accuracy. You may consider lowering the temperature during testing to achieve better results.

**Comment by XiaohuJoshua** [comment]
> Thank you for your attention! At present, we have not fine-tuned the models for the moment retrieval task on QVHighlights; we have only tested them for the highlight detection task. As our schedule is quite packed lately, we will fine-tune the models at a later time. Please stay tuned for updates.
> 
> By the way, we have discovered that reducing the temperature significantly improves the grounding accuracy. You may consider lowering the temperature during testing to achieve better results.

Thanks for you reply.

**Comment by gyxxyg** [comment]
We have released a more powerful temporal grounding video language model, [TRACE](https://github.com/gyxxyg/TRACE). You might consider trying it out for improved performance.
